app/assets/javascripts/pages/scenario-show-page.js.coffee

```diff
@@ -0,0 +1,15 @@
+class @ScenarioShowPage
+  constructor:() ->
+    @changemodaltext()
+
+  changemodaltext: () ->
+    $('#disable_all').click ->
+      $('.modal-body').text 'Would you like to disable all agents?'
+      $('#disabledfield').val 1
+    $('#enable_all').click ->
+      $('.modal-body').text 'Would you like to enable all agents?'
+      $('#disabledfield').val 0
```

Comment:
Is it alright to have two functions within one instance property? I was looking at `agent-show-page.js.coffee` and it seems to be valid.